common/api-review/firestore-lite.api.md

```diff
@@ -132,8 +137,8 @@ export class Firestore {
 // @public
 export interface FirestoreDataConverter<T> {
     fromFirestore(snapshot: QueryDocumentSnapshot<DocumentData>): T;
-    toFirestore(modelObject: T): DocumentData;
-    toFirestore(modelObject: Partial<T>, options: SetOptions): DocumentData;
+    toFirestore(modelObject: WithFieldValue<T>): DocumentData;
+    toFirestore(modelObject: NestedPartial<T>, options: SetOptions): DocumentData;

```

Comment:
I thought we were going to align those two names. Have you thought of a naming pattern that would achieve that?